packages/core-contracts/contracts/proxy/BeaconStorage.sol

```diff
@@ -3,7 +3,7 @@ pragma solidity ^0.8.0;
 
 contract BeaconStorage {
     struct BeaconNamespace {
```

Comment:
This namespace will only be used by beacon proxies, e.g. SyntheticUSD, SyntheticETH, etc.

This is how they know which beacon to ask for their implementation.